lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -200,7 +200,11 @@ function dosomething_campaign_preprocess_hot_module(&$vars, &$wrapper) {
     $rb_percent = 0;
   }
 
-  if ($rb_percent <= 20) {
+  $override_copy = variable_get('dosomething_campaign_progress_copy_override');
+  if (isset($override_copy)) {
+    $vars['progress_copy'] = $override_copy;
+  }
+  else if ($rb_percent <= 20) {
```

Comment:
Since 'elseif' and 'else if' aren't evaluated exactly the same in all contexts, and it's 'elseif' on line 206 below, it probably makes sense to make this 'elseif' for consistency.

http://php.net/manual/en/control-structures.elseif.php